cmake/dependencies/AMReX.cmake

```diff
@@ -234,10 +234,10 @@ set(WarpX_amrex_src ""
     "Local path to AMReX source directory (preferred if set)")
 
 # Git fetcher
-set(WarpX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
+set(WarpX_amrex_repo "https://github.com/WeiqunZhang/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(WarpX_amrex_internal)")
-set(WarpX_amrex_branch "3aafa306ad820bbcc50a4b7c14fe912406427d1d"
+set(WarpX_amrex_branch "fbsync"
```

Comment:
TODO: open as PR to mainline AMReX.

@RemiLehe saw the performance is good with this feature (no overhead in `FillBoundary` and saves the extra blocks to sync we saw in traces).

Reply:
I think the PR was already opened and merged in AMReX: https://github.com/AMReX-Codes/amrex/pull/2683.